docs/git-io-deprecation.md

```diff
@@ -39,44 +39,44 @@ or a `src` (or similar) directory under which you have all your `git` repos (dir
 ### Finding affected files
 
 Use the following command to find all occurrences in all directories recursively:
-```
+```bash
 grep -l "git\.io/build-harness" -R .
 ```
 Note that the above command can reach very deeply, such as into Terraform modules you have downloaded. You may want to impose some limits.
 If you run from the top level of a `git` repo, where there is a `Makefile` and a `Dockerfile`, you can reduce that to
-```
+```bash
 grep -l "git\.io/build-harness" *
 ```
 If you have a lot of Cloud Posse projects under a single directory, then you might try
-```
+```bash
 grep -l "git\.io/build-harness" * */*
 ```
 or for full depth below the current directory
-```
+```bash
 find . \( -name .terraform -prune -type f \)  -o \( -name build-harness -prune -type f \) -o \( -name 'Makefile*' -o -name 'README*' \) -type f
 ```
 
 ### Updating the affected files
 
 Once you are happy with the command to generate the list of files to update, update the files by inserting that command into this command template:
-```
+```bash
 sed -i '' 's/git.io\/build-harness/cloudposse.tools\/build-harness/' $(<command to list files>)
 ```
 
-#### Examples
+#### Samples
```

Comment:
Where did this change come from? I don't know what was wrong with "Examples" but I really don't like "Samples". How about "Options"?
```suggestion
#### Options
```

Reply:
@Nuru ^